File: 11/CG_IntersectHalfplanesDll/Misc/Line.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CG_IntersectHalfplanesDll.Primitives;

namespace CG_IntersectHalfplanesDll.Misc {
    public class Line {
        public Point left;
        public Point right;
        public bool reversed;

        public Line(Point l, Point r) {
            left = l;
            right = r;
        }
    }
}

File: compgeometry/src/ComputationalGeometry/misc/Line.java
package ComputationalGeometry.misc;

import ComputationalGeometry.Primitives.Point;

/**
 * Created by IntelliJ IDEA.
 * User: alex
 * Date: 2/28/13
 * Time: 2:15 PM
 */
public
class Line {
    public Point left;
    public Point right;
    public boolean reversed;

    public Line(Point l, Point r) {
        left = l;
        right = r;
    }
}
